language/Russian/strings.po

```diff
@@ -2320,29 +2346,29 @@ msgstr "Изменить папку"
 #, c-format
 msgctxt "gfxview"
 msgid "Zoom = 1/%1$d"
-msgstr ""
+msgstr "Масштаб = 1/%1$d"
 
 #: src/frontend/mame/ui/viewgfx.cpp:790 src/frontend/mame/ui/viewgfx.cpp:795
 #, c-format
 msgctxt "gfxview"
 msgid "Zoom = %1$d"
-msgstr ""
+msgstr "Масштаб = %1$d"
 
 #: src/frontend/mame/ui/viewgfx.cpp:800
 msgctxt "gfxview"
 msgid "Expand to fit"
-msgstr ""
+msgstr "Вместить"
```

Comment:
This translation may be misleading as it will only expand small tilemaps to fill the display – it will not shrink tilemaps that are too large to fit.  That’s why the English specifically says “Expand to fit”.